gfw_pixetl/tiles/vector_src_tile.py

```diff
@@ -135,23 +137,19 @@ def fetch_data(self) -> None:
         engine = create_engine(db_url)
 
         val_column = literal_column(str(self.layer.calc))
+        # geom_column = literal_column(GEOMETRY_COLUMN)
         geom_column = literal_column(str(self.intersection_geom()))
 
-        # Rename "geom" column to "WKT" as that's what gdal_rasterize
-        # looks for.
-        # gdal_rasterize can take "-oo GEOM_POSSIBLE_NAMES=geom"
-        # in GDAL 3.7+, which we're not on yet. We can use the column
-        # names as-is once we are and instead modify the
-        # gdal_rasterize command in self.rasterize()
         sql = (
-            select([val_column.label(self.layer.field), geom_column.label("WKT")])
+            select([val_column.label(self.layer.field), geom_column.label(GEOMETRY_COLUMN)])
             .select_from(self.src_table())
             .where(self.intersect_filter())
             .order_by(self.order_column(val_column))
         )
 
-        dataframe = pd.read_sql(sql, engine)
-        dataframe.to_parquet(dst, compression="snappy")
+        geodataframe = geopandas.read_postgis(sql, engine)
+        geodataframe.set_crs("EPSG:4326")
+        geodataframe.to_parquet(dst, compression="snappy")
```

Comment:
Can you add a comment here, about why you are using parquet?  Is it because it is faster or because it compresses better, or what?

Reply:
TBH it could be any of a handful of formats. We're just dumping it into a local file temporarily. However of the options (GeoPackage, Shapefiles, GeoJSON, CSV...) geoparquet is both fast and compact, without needing additional compression. I will add a comment to that effect.

Reply:
Done!